packages/common/src/types/field.ts

```diff
@@ -103,7 +119,12 @@ export type TableCalculation = {
 };
 
 export const isTableCalculation = (item: Item): item is TableCalculation =>
-    item ? !!item.sql && !('type' in item) && !('tableName' in item) : false;
+    item
+        ? !('binType' in item) &&
```

Comment:
Potential things to refactor: 

This type checking is not very reliable. I had to add `!('binType' in item)` after I added CustomDimension to Item becuase it was getting confused